### PR TITLE
Replace issue templates with issue forms [SDK-4168]

### PR DESCRIPTION
**Pull request by Widcket**
### Changes

This PR replaces the Markdown-based issue templates with [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms). These forms are a direct conversion of the Markdown issue templates, with the addition of a checklist at the top containing links to self-serve resources.

### Checklist

- [X] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
- [X] I have read the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
- [X] All existing and new tests complete without errors

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 blank_issues_enabled: false
 contact_links:
   - name: Auth0 Community
-    url: https://community.auth0.com/c/sdks/auth0js/59
```
Comment (Widcket) [inline .github/ISSUE_TEMPLATE/config.yml:4]: The path `/c/sdks/auth0js/59` no longer exists.

**Review by stevehobbsdev — CHANGES_REQUESTED** [review]
Please include the question "Which browsers have you tested in", which is on the original bug report. Not sure if it would be too restrictive to have a drop-down list with the major ones in case they're using something obscure, will leave that up to you.

**Comment by Widcket** [comment]
> Please include the question "Which browsers have you tested in", which is on the original bug report. Not sure if it would be too restrictive to have a drop-down list with the major ones in case they're using something obscure, will leave that up to you.

Sure thing, added in https://github.com/auth0/auth0.js/pull/1302/commits/4b32d2cd1de0751dd956473d5af3ef45c223558a (I used the browsers in https://gs.statcounter.com/browser-market-share/desktop/worldwide). Copy-paste mishap, because this field is not present in another browser library I did previously: https://github.com/auth0/lock/blob/master/.github/ISSUE_TEMPLATE/report_a_bug.md#environment.

**Comment by stevehobbsdev** [comment]
No worries, thanks. I think that change can/should be present in all the browser-based JS libraries (including Lock) but I can help roll it out as I see them 👍🏻

**Review by stevehobbsdev — APPROVED** [review]

**Comment by Widcket** [comment]
Added it to the lock PR as well: https://github.com/auth0/lock/pull/2324

**Pull request merged**